pages.es/common/vboxmanage-movevm.md

```diff
@@ -0,0 +1,12 @@
+# vboxmanage-movevm
```

Comment:
Thanks for your contribution I have a minor suggestion, will fix the same on the English page here.
```suggestion
# vboxmanage movevm
```